phpBB/language/en/email/test.txt

```diff
@@ -4,6 +4,10 @@ Hello {USERNAME},
 
 Congratulations. If you received this email, phpBB is correctly configured to send emails.
 
+<!-- IF MESSAGE -->
+{MESSAGE}
+
+<!-- ENDIF -->
```

Comment:
I guess Twig syntax can be used in emails too.

Reply:
It can as I have tested that. 
Not sure if we want it though, as literally everything in the language files is still phpbb syntax. I don’t know if translators are comfortable with it and know how to use it. They are not style developers after all. 
Perhaps it’s an idea to do all language files at once then rather then piece by piece.

Reply:
« Perhaps it’s an idea to do all language files at once then rather then piece by piece. » +1